### PR TITLE
Fix ticket #2711

### PR DESCRIPTION
**Pull request by weyert**
Potential fix ticket #2711

### CODE REVIEW

File: plugins/apps/subcommands/clone [hunk 1]
```diff
@@ -21,7 +21,9 @@ apps_clone_cmd() {
   local NEW_CACHE_DIR="$DOKKU_ROOT/$NEW_APP/cache"
 
   apps_create "$NEW_APP"
-  cp -a "$DOKKU_ROOT/$OLD_APP/." "$DOKKU_ROOT/$NEW_APP"
+  cd "$DOKKU_ROOT/$OLD_APP/."
```
Comment (josegonzalez) [inline plugins/apps/subcommands/clone]: Use pushd and popd instead of cd.

**Comment by weyert** [comment]
Hmm, why is it failing?

**Comment by josegonzalez** [comment]
Check how to run tests locally, it's documented. You just need shellcheck.

**Comment by weyert** [comment]
I will have a closer look tomorrow :)

**Comment by weyert** [comment]
Who is that the fix? I was still looking into it :)

**Pull request merged**